### DISCUSSION TITLE
supplementary variables in cubes after preprocessing functions

### DISCUSSION THREAD

**Discussion by rebeccaherman1** [ESMValGroup/ESMValCore#2161] [Q&A]
I am using [this notebook](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/notebooks/loading-and-processing-data.html) as a guide while I attempt to develop my recipe and diagnostic scripts. Upon loading the initial cube, I believe I see that the supplementary variable `areacella` included in the cube as `cell_area` under `Cell measures`, but the moment I restrict the domain with `extract_region`, `Cell measures` disappears, taking `cell_area` with it. Is areacella no longer part of the cube? If it is, where did it go?

**Comment by bouweandela** [comment]
Fixed by https://github.com/ESMValGroup/ESMValCore/pull/2166, which should be available in the upcoming v2.10 release.
